client/webserver/site/src/html/wallets.tmpl

```diff
@@ -719,48 +719,48 @@
         </tbody>
       </table>
       <p class="mt-3">
-        <label for="customVspUrl">Or add custom VSP URL:</label>
+        <label for="customVspUrl">[[[Or add custom VSP URL:]]]</label>
       </p>
        <div class="d-flex align-items-stretch w-100">
         <input 
           id="customVspUrl"
           type="url" 
           class="flex-grow-1"
           placeholder="https://" 
-          aria-label="Custom VSP URL"
+          aria-label="[[[Custom VSP URL]]]"
         >
-        <button id="customVspSubmit" class="btn btn-outline-secondary" type="button">Add</button>
+        <button id="customVspSubmit" class="btn btn-outline-secondary" type="button">[[[Add]]]</button>
       </div>
     </form>
 
     {{- /* PURCHASE TICKETS */ -}}
     <form id="purchaseTicketsForm" class="flex-center flex-column p-2 m-4 mw-425">
       <div class="form-closer hoverbg"><span class="ico-cross"></span></div>
-      <div class="flex-center fs22 sans-light">Purchase Tickets</div>
+      <div class="flex-center fs22 sans-light">[[[Purchase Tickets]]]</div>
       <div class="d-flex pt-2">
         <div class="flex-center flex-column col-50 p-4">
-          <span>Current Price</span>
+          <span>[[[Current Price]]]</span>
           <hr class="dotted my-1 w-100">
           <span id="purchaserCurrentPrice">a</span>
         </div>
         <div class="flex-center flex-column  col-50 p-4">
-          <span>Available Balance</span>
+          <span>[[[Available Balance]]]</span>
           <hr class="dotted my-1 w-100">
           <span id="purchaserBal">a</span>
         </div>
       </div>
       <hr class="dotted w-100">
       <div class="d-flex w-100 flex-wrap">
         <div class="flex-center flex-column col-12 py-2 brdrright">
-          <span>How Many?</span>
+          <span>[[[How Many?]]]</span>
```

Comment:
The regex we use for parsing these doesn't accept question marks. https://github.com/decred/dcrdex/blob/d353aad29f9119a782e718e8dba300c2ee03980d/client/webserver/locales/parse.go#L5
This one isn't resolving.

Reply:
Sorry about that, sorted.